src/components/AddItemForm/AddItemForm.js

```diff
@@ -19,8 +28,55 @@ const AddItemForm = ({ listId }) => {
   const handleSubmit = async (event) => {
     event.preventDefault();
 
```

Comment:
Something I noticed when testing my own PR's form: it's important to clear any old errors when the form is submitted for accessibility reasons.

If a user tries to submit the same bad input twice in a row, and you don't clear the error, there's no feedback for a visually-impaired user after the first time. Resetting the error message here would work really well with my later suggestion to add in an aria live region.
```suggestion
setErrorMessage('');

```